src/Cake.Common/Tools/OpenCover/OpenCoverHideSkippedOption.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Cake.Common.Tools.OpenCover
+{
+    using System;
+
+    /// <summary>
+    /// Represents the hide skipped option for OpenCover.
+    /// </summary>
+    [Flags]
+    public enum OpenCoverHideSkippedOption
+    {
+        /// <summary>
+        /// Removes information that skipped due to switch <value>-excludebyfile </value> from output file.
```

Comment:
Small typo in all of them, maybe rephrase to something like:
`Removes information from output files that relates to classes/modules that have been skipped (filtered) due to use of the <xxx>-switch.`
This would be in-line with the description in opencover.

Reply:
Updated.